sys-apps/polkit/polkit.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3.6
+# -*- coding: utf-8 -*-
+
+import os
+import stdlib
+import stdlib.split.drain_all
+from stdlib.template import autotools
+from stdlib.template.configure import configure
+from stdlib.patch import patch
+from stdlib.manifest import manifest
+
+
+def patch_polkit():
+    # There should be a dedicated user and group to take control of the polkitd daemon after it's started
+    stdlib.cmd("groupadd -fg 27 polkitd &&                          \
+        useradd -c 'PolicyKit Daemon Owner' -d /etc/polkit-1 -u 27  \
+        -g polkitd -s /bin/false polkitd                            \
+    ")
+    patch("-Np1 -i ./polkit-0.116-fix_elogind_detection-1.patch")
```

Comment:
Always use `stdlib.patch.patch_all()` instead of applying patches by hand ;)

(Btw, this patch is to use elogind instead of systemd-logind. Yet, we are using systemd-logind, so the patch isn't useful).